README.md

```diff
@@ -28,6 +28,8 @@ Supported password/key types:
   * `ec521` - generates ECC P-521 private key
   * `rsa2048` - generates 2048-bit RSA private key
   * `rsa4096` - generates 4096-bit RSA private key
+  * `x25519` - generates x25519 (also known as curve25519) ECC private key
+  * `ed25519` - generates ed25519 ECC private key
```

Comment:
Please also add this information to `gokey.1.md`.

Reply:
I think I did, didn't I? I see changes to `gokey.1.md` in my diff

Reply:
Sorry. You are right. I overlooked it.